GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -2694,17 +2694,18 @@ private void ViewFileHistoryMenuItem_Click(object sender, EventArgs e)
 
         private void Message_KeyUp(object sender, KeyEventArgs e)
         {
-            // Ctrl + Enter = Commit
             if (e.Control && e.KeyCode == Keys.Enter)
             {
-                ExecuteCommitCommand();
+                // Ctrl + Enter = Commit
+                // Ctrl + Shift + Enter = Commit and Push
+                ExecuteCommitCommand(e.Shift);
```

Comment:
Using a named parameter here will make it easier to read.

```suggestion
                ExecuteCommitCommand(push: e.Shift);
```

Reply:
Sorry, I missed that in your original review.  I'll add the named parameter.  I like the idea of the tooltip, however I'm not familiar with how GitExtensions is doing I18N.  Would it be acceptable for the tooltip to  say Ctrl + Enter and Ctrl + Shift + Enter, or do these need to be translatable strings?

Reply:
![image](https://user-images.githubusercontent.com/1251016/59401630-f6e39e80-8d60-11e9-9b17-5feecd6bad43.png)
![image](https://user-images.githubusercontent.com/1251016/59401649-06fb7e00-8d61-11e9-975a-08eb2f0c45e3.png)

Reply:
Looks good. I guess these would be translated. `TranslationString` is your friend. It should be easy to add.